site/contents/write_ups/2022/09-03-22_writeup.md

```diff
@@ -0,0 +1,95 @@
+
+If we are to have people in positions to bring such destruction and devastation into the world, we need those people to have a clear understanding of the moral implications of their decisions. Someone like [Stanislav Petrov](https://en.wikipedia.org/wiki/Stanislav_Petrov) is often credited as having "saved the world".
+
+## Who do you think would be held responsible for the implementation of a lethal autonomous weapon? 
+
+Maybe we would have fewer and fewer software developers? We don't want to encourage people to come into my profession (like some soldiers). This is already happening "Project Maven" DOD Project. The remit of this project was to develop certain advanced technology for defensive capabilities (e.g. developing computer vision software for identifying targets for drones). The company that initially won the contract was Google. Widespread protests which lead to Google ethical projects. Most developers didn't want to do this. DOD gave the contract to Palantir (tech company that mostly do government contracts; tracking systems, military applications etc).
+
+One of the problems with the free market means that someone will always do this job (same with like garbage collectors etc.) How do you uninvent these things? Evidence that CBRN weapons have been reduced in usage so maybe there is hope? Even if someone people will still produce them, other regulations will reduce the demand. (we're a good 5-20 years from the demand dropping for AI warfare, it's just such a cool gizmo to show off)
+
+## What change would you like to see on the basis of this piece? Who has the power to make that change?
+
+How necessary/unavoidable is war and in what sense? 
+
+It could be a question of when is it okay for a computer to make a decision versus a human: for accountability reasons it should be a human who makes decisions about e.g. employment or killing someone. Various pieces of media examine this idea in a more general setting. Some ask: [If we cannot hold computers accountable, should we be allowing them to make management decisions?](https://mobile.twitter.com/USC_ISI/status/1486770984443072516/photo/1). Others might ask, based on the way we make decisions: [Are You a Robot?](https://anchor.fm/are-you-a-robot/episodes/S7E1-Killer-Robots--Wanda-Muoz--Richard-Moyes-e12op43).
+
+We were disappointed that the legal aspect was treated by the speaker as if it's essentially a solved problem: if there was criminal intent then it will prosecuted that way, or criminal negligence it would be settled the other day. Plausible deniability/anonymity of drones is one possible attractive element to them, in a similar manner to how algorithms are sometimes used to deflect accountability when making decisions.
```

Comment:
```suggestion
We were disappointed that the legal aspect was treated by the speaker as if it's essentially a solved problem: if there was criminal intent then it will prosecuted that way, or criminal negligence it would be settled. Plausible deniability/anonymity of drones is one possible attractive element to them, in a similar manner to how algorithms are sometimes used to deflect accountability when making decisions.
```